tigramite/pcmci.py

```diff
@@ -761,12 +757,12 @@ def _set_max_condition_dim(self, max_conds_dim, tau_min, tau_max):
 
         Returns
         -------
-        max_cond_dim : int
+        max_conds_dim : int
             Input maximum condition dimension or default
         """
         # Check if an input was given
         if max_conds_dim is None:
-            max_conds_dim = self.N * (tau_max - tau_min + 1)
```

Comment:
remove +1 here, make sure that max_conds_dim = 2 iterates through 0, 1, 2 in main loop and wherever similar concept is used.

Reply:
done